### DISCUSSION TITLE
Any way to easily split existing models into model-0000X-of-0000X format?

### DISCUSSION THREAD

**Discussion by Slug-Cat** [oobabooga/text-generation-webui#608] [Q&A]
My computer kind of dies for a bit trying to allocate enough RAM for the entire Llama 13B model in one go, but has no issues with other models in a split format. Is there any way to convert an existing monolithic model into smaller pieces for easier loading?

**Comment by oobabooga** [comment]
You can use this script, it converts to safetensors and reshards https://github.com/oobabooga/text-generation-webui/blob/main/convert-to-safetensors.py
